package.json

```diff
@@ -7,9 +7,12 @@
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
+    "react-carousel-responsive": "^0.0.31",
```

Comment:
There is 2 packpages for the same thing, please delete the one that are you not using